Directory.Build.props

```diff
@@ -3,15 +3,15 @@
     <MicrosoftSqlServerCompactPackageVersion>4.08876.1</MicrosoftSqlServerCompactPackageVersion>
     <MicrosoftVisualStudioComponentModelHostPackageVersion>17.0.32-g9d6b2c6f26</MicrosoftVisualStudioComponentModelHostPackageVersion>
     <MicrosoftVisualStudioExtensibilityHostingPackageVersion>17.0.32-g9d6b2c6f26</MicrosoftVisualStudioExtensibilityHostingPackageVersion>
-    <MicrosoftVisualStudioPackageCurrent>17.0.0-preview-2-31226-057</MicrosoftVisualStudioPackageCurrent>
+    <MicrosoftVisualStudioPackageCurrent>17.0.0-previews-2-31405-002</MicrosoftVisualStudioPackageCurrent>
```

Comment:
I guess the build will either pass or it won't, but boy does the `s` here look like a typo 😛

Reply:
Yeah, I thought the same thing but we double-checked it and it did build 🤷.